app/Http/Controllers/Api/HetznerController.php

```
@@ -0,0 +1,651 @@
+                \App\Actions\Server\ValidateServer::dispatch($server);
+            }
+
+            return response()->json([
+                'uuid' => $server->uuid,
+                'hetzner_server_id' => $hetznerServer['id'],
+                'ip' => $ipAddress,
+            ])->setStatusCode(201);
+        } catch (\Throwable $e) {
+            return response()->json(['message' => 'Failed to create server: '.$e->getMessage()], 500);
+        }
+    }
```

PHPMD (2.15.0)

The method createServer() has a Cyclomatic Complexity of 26. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)